### PR TITLE
Simple implementation of fortify for rasters

### PR DESCRIPTION
**Pull request by Rekyt**
As suggested in #125 here is the corresponding PR ;)

### CODE REVIEW

**Comment by terrytangyuan** [comment]
Could you add a test and attach an example plot here?

**Comment by sinhrks** [comment]
Thx for the PR:) I'm fixing failure tests unrelated to the PR. sorry for the trouble.

**Comment by Rekyt** [comment]
Alright, I'll add the tests ;)

**Comment by Rekyt** [comment]
Here they are, please tell me if I need to change something else.

**Comment by terrytangyuan** [comment]
A few comments:

- You need to add `raster` to suggest field in DESCRIPTION and travis script [here](https://github.com/sinhrks/ggfortify/blob/master/.travis.yml#L9). 
- Do you need an associated `autoplot` function for those fortified objects? Could you paste some example plot here?

**Comment by sinhrks** [comment]
Also pls rebase to the master, as I've fixed some test failures.

**Comment by sinhrks** [comment]
Thx for the rebase. 

- Pls fix the failure tests. Also it's nice to add ``autoplot`` tests, at least it shouldn't raise.
- Pls add description to ``README``

**Comment by Rekyt** [comment]
Wow, I don't think the commit history will be very readable. But I added tests and autoplot methods.
```
r <- raster(nrows = 10, ncols = 10, xmn = 0, xmx = 10, ymn = 0, ymx = 10)
r[] <- 0

autoplot(r)
```
Gives the following:
rplotggfortify

**Comment by terrytangyuan** [comment]
It's a pure blue plot. Is that intentional?

**Comment by Rekyt** [comment]
Yes. Because all the raster values are equal to zero.
This is what you get when it's not the case:
```
library(raster)
set.seed(1)
r <- raster(nrows = 10, ncols = 10, xmn = 0, xmx = 10, ymn = 0, ymx = 10)
r[] <- sample(100, 100, replace = T)
autoplot(r)
```
image

**Comment by terrytangyuan** [comment]
Awesome! Could you rebase and resolve conflicts?

**Comment by terrytangyuan** [comment]
@Rekyt Try a `git fetch upstream master` and then `git pull --rebase upstream master`. Seems like you are rebasing to an out-dated upstream head.

File: man/ggmultiplot-class.Rd [hunk 1]
```diff
@@ -2,23 +2,32 @@
...
+\alias{[[<-,ggmultiplot,ANY,ANY-method}
 \alias{[[<-,ggmultiplot-method}
 \alias{ggmultiplot-class}
 \alias{length,ggmultiplot-method}
 \title{An S4 class to hold multiple \code{ggplot2::ggplot} instances}
 \usage{
 \S4method{length}{ggmultiplot}(x)
 
-\S4method{[}{ggmultiplot}(x, i, j, ..., drop = TRUE)
+<<<<<<< HEAD
```
Comment (terrytangyuan) [inline man/ggmultiplot-class.Rd]: Unresolved here.

**Comment by terrytangyuan** [comment]
@Rekyt Thank you for the contribution! As mentioned by @sinhrks, this will be included as part of the next release to CRAN. Going forward, it would be good to have some vignettes like for other packages. It would also be good to compare with `RasterViz` to see what `ggfortify` is lack of.

**Comment by Rekyt** [comment]
You're welcome ;) Yep, or at least good examples showing its uses. Sorry for the very messy history, I didn't thought I would have been that complex.

**Pull request merged**